lib/jekyll-plugin-platoniq-journal/numbers_tag.rb

```diff
@@ -47,21 +47,21 @@ def include_file_path
     def render_all
       output = []
 
-      output << %(<ul class="pj-numbers">)
+      output << %(<dl class="pj-numbers">)
```

Comment:
should this be a `<div>`?
```suggestion
      output << %(<div class="pj-numbers">)
```